sdk/nodejs/provisioning/bin.ts

```diff
@@ -36,7 +36,8 @@ export class Bin implements EngineConn {
   private readonly cacheDir = path.join(
     `${
       process.env.XDG_CACHE_HOME?.trim() || envPaths("", { suffix: "" }).cache
-    }/dagger`
+    }`,
+    "dagger"
```

Comment:
@slumbering So there actually was a problem using the forward slash in Windows?

Reply:
I thought it was changed as part of you suggestions ? 🤔

Reply:
He added the `path.join()` but suffixed the path with `/dagger` instead of passing as argument to `.join(..., "dagger")` like this change is doing.